app/views/build_estimates/employment.html.slim

```diff
@@ -0,0 +1,24 @@
+- content_for :back do
+  = link_to t("generic.back"), previous_wizard_path, class: "govuk-back-link"
+
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    span.govuk-caption-l = t(".caption")
+    h1.govuk-heading-xl = t(".heading")
+    span.govuk-caption-m class="govuk-!-padding-bottom-4" = t(".second_caption")
+    = form_for(@form, url: wizard_path, method: :put) do |form|
+      = form.govuk_error_summary t("generic.error_summary_title")
+
+      - EmploymentForm::DECIMAL_ATTRIBUTES.each do |field|
+        = render "money_input",
+                  form:,
+                  field:,
+                  width: 5,
+                  label_text: t(".#{field}.input")
+      = form.govuk_collection_select :frequency,
+                                     @form.frequency_options,
```

Comment:
Is there any specific reason for adding the blank into the options, rather than passing blank: true to govuk_collection_select (it forwards to collection_select)?

Reply:
Just my own ignorance! Fixing.